src/host/screenInfo.cpp

```diff
@@ -1513,19 +1513,6 @@ NT_CATCH_RETURN()
     auto& gci = ServiceLocator::LocateGlobals().getConsoleInformation();
     auto status = STATUS_SUCCESS;
 
-    // If we're in conpty mode, suppress any immediate painting we might do
-    // during the resize.
-    if (gci.IsInVtIoMode())
-    {
-        gci.GetVtIo()->BeginResize();
-    }
-    auto endResize = wil::scope_exit([&] {
-        if (gci.IsInVtIoMode())
-        {
-            gci.GetVtIo()->EndResize();
-        }
-    });

```

Comment:
While working on another PR I realized that this special behavior is only here for `TextBuffer::Reflow`, because that one used to use `NewlineCursor` which calls `IncrementCircularBuffer` which calls `Renderer::TriggerFlush` which calls `VtEngine::InvalidateFlush` which then set `pForcePaint = false` if `BeginResize` was called.

In short: This block prevents `TextBuffer::Reflow` from setting `pForcePaint` to `true`.

...and the new code doesn't call `IncrementCircularBuffer` anymore so that's moot.